recipes/libcap/all/conanfile.py

```diff
@@ -64,8 +61,10 @@ def generate(self):
         env.define("lib", "lib")
 
         if cross_building(self) and not env.vars(self).get("BUILD_CC"):
-            native_cc = "cc"
-            self.output.info("Using native compiler '{}'".format(native_cc))
+            native_cc = VirtualBuildEnv(self).vars().get("CC")
```

Comment:
Nice trick! The global.conf has `compiler_executables` which can configure which compiler should be used, but it's automatically managed in Toolchain generators.

Reply:
where this BUILD_CC would come from? I dont think it's the env var to listen.

Reply:
@uilianries I don't think `BUILD_CC` is set automatically, is it? I used @SpaceIm's trick from the Boost recipe here, but I can't actually determine the `compiler_executables` in the build context, see issue #12860.

Reply:
BUILD_CC is not a standard env variable